src/tests/tablet/schedule/AMeetingCannotBeCreatedWithoutAttendees.java

```diff
@@ -0,0 +1,45 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC10: Verify a meeting cannot be created if the attendees are not specified
+ * @author Asael Calizaya
+ *
+ */
+public class AMeetingCannotBeCreatedWithoutAttendees {
+
+	@Test(groups = "ACCEPTANCE")
+	public void testAMeetingCannotBeCreatedWithoutAttendees() {
+		ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+		List<Map<String, String>> meetingData  = excelReader.getMapValues("MeetingData");
+		String organizer = meetingData.get(0).get("Organizer");
+		String subject = meetingData.get(0).get("Subject");
+		String startTime = meetingData.get(0).get("Start time");
+		String endTime = meetingData.get(0).get("End time");
+		String body = meetingData.get(0).get("Body");
+
+		HomeTabletPage homeTabletPage = new HomeTabletPage();
+		SchedulePage schedulePage = homeTabletPage.clickScheduleBtn();
+		schedulePage.setOrganizerTxtBox(organizer)
+		.setSubjectTxtBox(subject)
+		.setStartTimeDate(startTime)
+		.setEndTimeDate(endTime)
+		.setBodyTxtBox(body)
```

Comment:
The steps 36, 37, 38 are not necessary, please check this.... you should omit this steps